tests/filters_test.rs

```diff
@@ -495,6 +495,21 @@ fn can_sum() {
     assert_eq!(actual, "203")
 }
 
+#[test]
+fn can_average() {
+    let actual = nu!(
+        cwd: "tests/fixtures/formats", h::pipeline(
+        r#"
+            open sgml_description.json
+            | get glossary.GlossDiv.GlossList.GlossEntry.Sections
+            | average
+            | echo $it
+        "#
+    ));
+
+    assert_eq!(actual, "101.5000000000000")
+}
+
```

Comment:
:)

Reply:
I am going to add one more test for `Bytes` and then I think I'll take this out of draft status.. @andrasio - I was looking at those examples last night and decided I do not need to do any string conversions as it can be converted by the user (maybe I can add a help message?). Thank you for your help!

Reply:
So far we use the commands Signature for displaying when using `help`. At some point we will include longer help messages and examples of usage. If you like, and would like to add documentation for `average` [please read here](https://github.com/nushell/nushell/issues/711).

Reply:
I wonder if the test failure is happening because directories have a different size in Windows. Maybe it would work if you filter out directories, and then average what's left?

Reply:
I'll check it out on my Windows machine. If that is the case, I'll add it to the help page and see if it effects `sum` as well.